docs/README.md

```diff
@@ -0,0 +1,118 @@
+# Pillarbox Web Suite
+
+![Pillarbox logo](README-images/logo.jpg)
+
+This repository is a comprehensive suite of plugins, themes, and components designed to boost the
+video player experience of the [pillarbox-web](https://github.com/SRGSSR/pillarbox-web) player.
+
+These tools provide versatile options for customization and enhancement tailored to meet your needs.
+
+## Requirements
+
+To use the components in this suite, you need the following installed on your system:
+
+- Node.js
+
+## Quick Start
+
+To get started with this suite, follow these steps:
+
+Add the `@srgssr` registry to your `.npmrc` file:
+
+```plaintext
+//npm.pkg.github.com/:_authToken=TOKEN
+@srgssr:registry=https://npm.pkg.github.com
+```
+
+Generate an authentication token by following this
+guide: [Authenticating with a personal access token](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-with-a-personal-access-token)
+
```

Comment:
Could we also add the direct link to generate a [token](https://github.com/settings/tokens) ?

This is something we can also improve in the main repo. The other day, I had to generate a new token and ended up following the steps described in the Letterbox repo.

Reply:
Modified the sentence so that the settings link appears first.